src/Guardrails-Common/GR-Common.psm1

```diff
@@ -436,34 +456,41 @@ function Check-GAAuthenticationMethods {
     
     }
 
-    # GA UPN list has less than 2 UPN
+    # Condition: GA UPN list has less than 2 UPN
     if ($globalAdminUPNs.Count -lt 2) {
         $commentsArray += $msgTable.globalAdminMinAccnts
     }
-    # GA UPN list has > 2 UPNs and all are MFA enabled
+    # Condition: GA UPN list has > 2 UPNs and all are MFA enabled
     elseif($globalAdminUPNs.Count -ge 2 -and $mfaCounter -eq $globalAdminUPNs.Count) {
         $commentsArray += $msgTable.globalAdminMFAPassAndMin2Accnts
         $IsCompliant = $true
     }
-    # GA UPN list has > 2 UPNs and not all UPNs are MFA enabled
+    # Condition: GA UPN list has > 2 UPNs and not all UPNs are MFA enabled
     else{
-        # only one UPN is not MFA enable
-        if ( $GAUPNsMFA.Count -eq 1 ) {
-            $commentsArray += $msgTable.globalAdminAccntsMFADisabled1 -f $GAUPNsMFA.MFAComments
-        }
-        # None are MFA enabled
-        elseif ( $GAUPNsMFA.Count -eq $globalAdminUPNs.Count) {
-            $commentsArray += $msgTable.globalAdminAccntsMFADisabled3
+        # This will be used for debugging
+        if($GAUPNsMFA.Count -lt 1){
```

Comment:
Its the same thing but I think this makes more sense i.e its an empty array
`$GAUPNsMFA.Count -eq 0`

Reply:
True :)

Reply:
I second that